src/components/QuestionRender.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { hot } from 'react-hot-loader';
+import PropTypes from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+
+import {
+  Typography,
+  CardContent,
+  TextField,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  cardcontent: {
+    '&:last-child': { paddingBottom: 16 },
+  },
+  markdown: {
+    '& p': { marginBottom: 0, marginTop: 0, overflowWrap: 'anywhere' },
+  },
+  white: {
+    '&.cardroot': {
+      backgroundColor: 'white',
+    },
+    '&.noAnswer': {
+      opacity: '70%',
+      fontStyle: 'italic',
+    },
+  },
+  gray: {
+    '&.cardroot': {
+      backgroundColor: '#F9F9F9',
+    },
+    '&.noAnswer': {
+      opacity: '70%',
+      fontStyle: 'italic',
+    },
+  },
+  unfinishedRequired: {
+    color: 'red',
+    '&.cardroot': {
+      borderColor: 'red',
+    },
+  },
+});
+
+function QuestionRender({
+  answer, required, questionID, isReadOnly, editMode,
+  toggleEditMode, getClass, getRequiredClass, blogQuestions,
+}) {
+  const classes = useStyles();
+
+  function onChangeHandler(e) {
```

Comment:
Can this be changed to an arrow function to be consistent with the rest of the application?